<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentWindow</class>
 <widget class="QDialog" name="PaymentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="paymentTitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Payment</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentBalanceDisplay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentBalanceTitle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Balance:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showCurrentDataButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>461</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Current Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointsDisplay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentPointsTitle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Points:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cashRadioButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cash</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="creditRadioButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="balanceRadioButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QComboBox" name="creditCardsComboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>407</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mastercard    5123 4590 4605 8920    $15,982</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mastercard    5427 6600 6424 1339    $26,720</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="dialog1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>239</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I would like to pay using:</string>
   </property>
  </widget>
  <widget class="QLabel" name="creditCardTitle">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Credit Card:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="payButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Pay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>580</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountDueTitle">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount Due:</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountDueDisplay">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="paymentStatus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Payment Status:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
